Create bugreport.yml

csmir opened the pull request:
Grabbed from [Discord.Net-Labs](https://github.com/Discord-Net-Labs/Discord.Net-Labs/blob/release/3.x/.github/ISSUE_TEMPLATE/bugreport.yml), original by @CottageDwellingCat.

csmir committed:
Create bugreport.yml

Grabbed from Discord.Net-Labs, original by @CottageDwellingCat

quinchs commented on .github/ISSUE_TEMPLATE/bugreport.yml:
```diff
@@ -0,0 +1,73 @@
+  - type: checkboxes
+    attributes:
+      label: Check The Docs
+      description: Please refer to our [FAQs](https://labs.discordnet.dev/faq), [Documentation](https://labs.discordnet.dev/api/index.html), 
```
```suggestion
      description: Please refer to our [FAQs](https://discordnet.dev/faq/basics/getting-started.html), [Documentation](https://discordnet.dev/api/index.html), 
```

quinchs commented on .github/ISSUE_TEMPLATE/bugreport.yml:
```diff
@@ -0,0 +1,73 @@
+    attributes:
+      label: Check The Docs
+      description: Please refer to our [FAQs](https://labs.discordnet.dev/faq), [Documentation](https://labs.discordnet.dev/api/index.html), 
+                    and [Migration Guide](https://labs.discordnet.dev/guides/v2_v3_guide/v2_to_v3_guide.html) before reporting issues.
```
```suggestion
                    and [Migration Guide](https://discordnet.dev/guides/v2_v3_guide/v2_to_v3_guide.html) before reporting issues.
```

quinchs commented on .github/ISSUE_TEMPLATE/bugreport.yml:
```diff
@@ -0,0 +1,73 @@
+    attributes:
+      label: Version
+      description: What version of dnet are you using?
+      placeholder: ex. 3.5.0-labs
```
```suggestion
      placeholder: ex. 3.1.0
```

csmir committed:
Append faq link

Co-authored-by: Quin Lynch <49576606+quinchs@users.noreply.github.com>

csmir committed:
Append migration guide

Co-authored-by: Quin Lynch <49576606+quinchs@users.noreply.github.com>

csmir committed:
Append version change

Co-authored-by: Quin Lynch <49576606+quinchs@users.noreply.github.com>

quinchs commented on .github/ISSUE_TEMPLATE/bugreport.yml:
```diff
@@ -0,0 +1,73 @@
+body:
```
```suggestion
name: 🐞 Bug Report
description: File a bug report
title: "[Bug]: "
labels: ["bug"]
body:
```

csmir committed:
Append body

Co-authored-by: Quin Lynch <49576606+quinchs@users.noreply.github.com>

quinchs commented on .github/ISSUE_TEMPLATE/bugreport.yml:
```diff
@@ -0,0 +1,77 @@
+    id: version
+    attributes:
+      label: Version
+      description: What version of dnet are you using?
```
```suggestion
      description: What version of Discord.Net are you using?
```

csmir committed:
Fix 5/???

smh I was tired when I committed this and didnt bother making changes assuming it was already good :'')

Co-authored-by: Quin Lynch <49576606+quinchs@users.noreply.github.com>
